serving/samples/helloworld-clojure/Dockerfile

```diff
@@ -13,9 +13,8 @@ COPY . .
 # Build an uberjar release artifact.
 RUN mv "$(lein uberjar | sed -n 's/^Created \(.*standalone\.jar\)/\1/p')" app-standalone.jar
 
-# Configure and document the service HTTP port.
+# Configure the service HTTP port.
```

Comment:
```suggestion
# Set the default value for PORT environment variable.
```

Reply:
Should this change be consistent across all the samples?

Reply:
Correct, just trying to figure out the phrasing.